#Publish all joint states
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

#Position Controller
#pid: {p: 10000.0, i: 10.0, d: 10.0}
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint1_joint
  pid: {p: 2500.0, i: 10.0, d: 10.0}
joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint2_joint
  pid: {p: 50.0, i: 10.0, d: 10.0}
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: joint3_joint
  pid: {p: 50.0, i: 10.0, d: 10.0}
gripper_main_position_controller:
  type: effort_controllers/JointPositionController
  joint: gripper_main_joint
  pid: {p: 50.0, i: 10.0, d: 10.0}
gripper_right_position_controller:
  type: effort_controllers/JointPositionController
  joint: gripper_right_joint
  pid: {p: 100000.0, i: 10.0, d: 10.0}
gripper_left_position_controller:
  type: effort_controllers/JointPositionController
  joint: gripper_left_joint
  pid: {p: 100000.0, i: 10.0, d: 10.0}